client-spark/spark-4-columnar-shuffle/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,33 @@
+#
```

Comment:
Spark moved to log4j2 since 3.3, so we don't need the log4j1 conf for Spark 4

Reply:
done